docs/assembly-control-unit.md

```diff
@@ -171,47 +281,105 @@ Heat  |  Stir |
-  - Insert the other ends of that cable into one of the 3-wide lever connectors. Each slot on the connector should accomodate 2 of the wires. Twisting the wires together with pliers may help. In the unused slot, insert one of the black wires - later we will connect it to the (-) side of the 1-to-3 lever connector that will act as the 12v power manifold.
-  - Insert each of the short wires into the controlled side of the relay in a terminal labeled "NO" (normally open). Screw the terminals down gently but firmly.
-  - Insert the corresponding ends of the 3-wire cable into the terminals labeled "COM" (common) and tighten the terminals.
-  - Insert the other ends of the 3-wire cable into the other 3-wide lever connector, again doubling up in one of the slots. In the remaining open slot, insert one of the short red wires, which will later connect to the 12v manifold.
+-Wiring the Phoneix connector
+  - Positives. On the back on the female 8-pin Phoneix connector, insert a 4" green, red, blue wire into the correct positive (+) terminal and tighten the terminals. The black terminal will be empty for now.
+      - The wire pattern will be reversed when adding wires from the the back of the female connector. So from left to right, skip the first two terminals and then place the green +, red + and blue + wires.
+
+      <IMG ALT="Cable order" SRC="./media/control-unit/wire_rev.png" WIDTH="200" />
+
+Note: When working with screw terminals, remember that they are only as secure as they are tight. Use firm but gentle turns when screwing down terminals - you want them to be tight, but you don't want to damage the slots on the heads of the screws. You should be able to lightly tug on the wires without them moving.
+
+  - Negatives. On the Phoenix connector, take the 6" 4-wire cable and insert each of its wires (black, green, red, blue) into the correct negative (-) terminal and screw them down
+  - Take the other end of the 4-wire cable and place the black and green wire in their own slots of the 4-to-1 wire connector. Then put the red and blue wires together in one slot. That leaves the last slot open for power (-12V), put the 4" black wire in that slot.
+
+  <IMG ALT="Wire the negative Phoenix wires and connector" SRC="./media/control-unit/neg12v.png" WIDTH="400" />
+
+
+  - Wiring the relay   
+
+    <IMG ALT="4-channel relay" SRC="./media/control-unit/relay.png" WIDTH="300" />
+
+     - Take the female phoneix connector and insert each of its 4" positive wires into the "NO" (Normally Open) terminals on the 4-channel relay board. The terminals are labeled "NO1", NO1 as so on. Screw the terminals down gently but firmly.
+        <IMG ALT="Wire the negative Phoenix wires and connector" SRC="./media/control-unit/pos12v.png" WIDTH="500" />
+     - Use the 3-wire cable, put each of its wires into the terminals labeled "COM" (common) and tighten the terminals.
```

Comment:
We need to call these the Positive and Negative cables probably (instead of 3- and 4- wire)